README.md

```diff
@@ -30,6 +30,17 @@ HTML-Proofer finished successfully.
 ```
 in the output
 
+### Additional notes:
+
+* If developing with docker machine/vagrant/local guest OS, you may find issues port forwarding port `4000` correctly as it listens on `127.0.0.1` on the guest OS. 
+* You may also notice (when mounting the project workspace into the docker container with `--volume=$(pwd):/srv/jekyll`) that file ownership permissions get messed up. 
+
+If you experience these, you can fix both of those with:
+
+```bash
+docker run --name istio-jekyll --volume=$(pwd):/srv/jekyll  -it -p 4000:4000 jekyll/jekyll:3.5.2 sh -c "bundle install && rake test && bundle exec jekyll serve --incremental --host 0.0.0.0"
```

Comment:
this seems a bit long winded, so you are saying —host helps with the network 

what flag helps with the permissions ?

we could just add those to the first command line

Reply:
`-p 4000:4000` for the docker run and `--host 0.0.0.0` for the `jekyll serve` help with the network


running `bundle exec jekyll serve` instead of just `jekyll serve` helped with the filesystems perms (something to do with how it runs the chown commands jekyll serve does when creating _site)

Reply:
happy to make it more pithy